config.toml

```diff
@@ -138,6 +138,15 @@ sidebar_menu_compact = true
   [[module.imports]]
     path = "github.com/matrix-org/docsy"
     disable = false
+  # Mounting /assets and /static as assets directories helps Hugo to generate
+  # the correct URL.
+  [[module.mounts]]
+    source = 'assets'
+    target = 'assets'
```

Comment:
Is this needed to ensure that `assets` remains mounted to `assets` (the default, but defaults are not kept when defining any `module.mounts` entries)? If so, could you leave a comment noting that? Otherwise I fear this may appear redundant and be accidentally removed.

Reply:
Yes, if we override a mount, Hugo ignores the default.

Reply:
Could you leave a comment noting that? :)

Reply:
This is not a problem anymore with the latest commits.